Pracs/prac_07/project_management.py

```diff
@@ -98,14 +101,22 @@ def add_project(projects):
     """Add a new project to the projects list."""
     print("Let's add a new project")
     name = input("Name: ")
```

Comment:
Should a blank name project be allowed?

Reply:
Oh I forgot to add error checking for the add function, missed it somehow, thanks!